tools/sgbiome/tools.go

```diff
@@ -0,0 +1,65 @@
+package sgbiome
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	toolName = "biome"
+	version  = "1.6.0"
+)
+
+func Format(ctx context.Context, paths ...string) error {
+	sg.Deps(ctx, prepareCommand)
+
```

Comment:
nit: avoid adding blank lines in small functions since where to put them is based on individual taste

Reply:
I always comment on this because different people have such different preferences around blank lines that the easiest thing is to keep them to a minimum (at least in small-ish functions).